### ISSUE TITLE
Update `opening_brace` for multiline if statements

### ISSUE THREAD

**Issue by Juice805** [realm/SwiftLint#3720]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule update request

Similar to #3068 I'd like to propose a change to `opening_brace` to allow the following format:

```swift
if let child = parent.children.first,
    let grandchild = child.children.first,
    let greatGrandchild = grandchild.children.first
{
    // Some code
}
```

or 
```swift
if 
    let child = parent.children.first,
    let grandchild = child.children.first,
    let greatGrandchild = grandchild.children.first
{
    // Some code
}
```

Currently the autocorrect will fix it to the following which is difficult to read:

```swift
if 
    let child = parent.children.first,
    let grandchild = child.children.first,
    let greatGrandchild = grandchild.children.first {
    // Some code
}
```

**Comment by Juice805** [comment]
looks like `while` also has this issue and should be included:

```swift
while 
    let child = parent.children.first,
    let grandchild = child.children.first,
    let greatGrandchild = grandchild.children.first
{
    // Some code
}
```

`guard` does not assuming this structure:

```swift
guard
    let child = parent.children.first,
    let grandchild = child.children.first,
    let greatGrandchild = grandchild.children.first
else {
    // Some code
}
```
